### PR TITLE
Feature/scrubbing

### PR DESCRIPTION
**Pull request by martinstannard**

### CODE REVIEW

File: Pages/Index.elm [hunk 1]
```diff
@@ -53,6 +58,7 @@ type alias Model =
     , settings : Settings
     , studentId : StudentId
     , mdl : Material.Model
+    , slider : Float
```
Comment (mordrax) [inline Pages/Index.elm]: let's make this a Int and call it 'sliderIndex' so it's not confused with the actual slider and round it on ln 124 when the SliderMsg is being saved.

**Comment by mordrax** [comment]
notwithstanding minor points to be considered and fixed after, 🚢 !!!

is that a correct usage of the phrase notwithstanding?

**Comment by martinstannard** [comment]
Yes, it is the correct usage

**Comment by JulianLeviston** [comment]
Wow... gotta be super fast around here to make a comment :)

**Comment by mordrax** [comment]
Too slow! What was your comment though? We can merge them into the next PR.

**Comment by JulianLeviston** [comment]
@mordrax oh it's too late now, but I would have liked to have just discussed making the slider not be a top-level entity as it only relates to when there's data on the page, but that's been pointed out lots already. I'm part way through a spike refactoring to how I'd like the architecture to be to see what people think, but not quite there yet. Hopefully we can not add too many features before I've finished a prototype of it.

**Pull request merged**